specification/communication/data-plane/PhoneNumbers/preview/2023-10-01-preview/phonenumbers.json

```diff
@@ -0,0 +1,2062 @@
+          "x-ms-enum": {
+            "name": "Error",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PurchasedPhoneNumber": {
+      "description": "Represents a purchased phone number.",
+      "required": [
+        "assignmentType",
+        "capabilities",
+        "cost",
+        "countryCode",
+        "id",
+        "phoneNumber",
+        "phoneNumberType",
+        "purchaseDate"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The id of the phone number, e.g. 11234567890.",
+          "type": "string"
+        },
+        "phoneNumber": {
+          "description": "String of the E.164 format of the phone number, e.g. +11234567890.",
+          "type": "string"
+        },
+        "countryCode": {
+          "description": "The ISO 3166-2 code of the phone number's country, e.g. US.",
+          "type": "string"
+        },
+        "phoneNumberType": {
+          "description": "The phone number's type, e.g. geographic, tollFree.",
+          "enum": [
+            "geographic",
+            "tollFree"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberType",
+            "modelAsString": true
+          }
+        },
+        "capabilities": {
+          "$ref": "#/definitions/PhoneNumberCapabilities"
+        },
+        "assignmentType": {
+          "description": "The assignment type of the phone number. A phone number can be assigned to a person, or to an application.",
+          "enum": [
+            "person",
+            "application"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberAssignmentType",
+            "modelAsString": true
+          }
+        },
+        "purchaseDate": {
+          "format": "date-time",
+          "description": "The date and time that the phone number was purchased.",
+          "type": "string"
+        },
+        "cost": {
+          "$ref": "#/definitions/PhoneNumberCost"
+        }
+      }
+    },
+    "PurchasedPhoneNumbers": {
+      "description": "The list of purchased phone numbers.",
+      "required": [
+        "phoneNumbers"
+      ],
+      "type": "object",
+      "properties": {
+        "phoneNumbers": {
+          "description": "Represents a list of phone numbers.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PurchasedPhoneNumber"
+          }
+        },
+        "nextLink": {
+          "description": "Represents the URL link to the next page of phone number results.",
+          "type": "string"
+        }
+      }
+    },
+    "PhoneNumbersBrowseRequest": {
+      "properties": {
+        "phoneNumberType": {
+          "description": "Represents the number type of the offering.",
+          "enum": [
+            "geographic",
+            "tollFree"
+          ],
+          "type": "string",
+          "x-ms-enum": {
```

Comment:
By GA, all enum values should be documented throughout. This goes under `x-ms-enum` and you can find documented under https://github.com/Azure/autorest under "docs".